contracts/Rivendale.sol

```diff
@@ -132,9 +153,19 @@ contract Rivendale {
                 voteBreakdown[_id].walkerTally++;
             }
         }
-        if (voteBreakdown[_id].walkerCount > 0){
-            weightedVotes[0] = weights.walkerWeight * (voteBreakdown[_id].walkerTally / voteBreakdown[_id].walkerCount);
+
+        // Since walkers are a priviledged group, shouldn't the tally be based on the total amount of walkers? If there's huge abstention, a single walker can have 40% of voting power.
+
+        // Follow up if above is implemented:
+        // If a waker is banished while a vote is open, will this calculation be messed up?
+        if (voteBreakdown[_id].walkerCount > 0) {
+            weightedVotes[0] =
+                weights.walkerWeight *
+                (voteBreakdown[_id].walkerTally /
+                    voteBreakdown[_id].walkerCount);
         }
+
+        // Can't rich members, in a way, pay to have larger weight passing as a payee? This could be an attack vector in periods of few paying customers

```

Comment:
you want that though.  If someone is paying your walkers....they sort of own them.  The checks come in the fact that the combo of TRB holders and walkers can vote against them